jekyll/_cci2/persist-data.md

```diff
@@ -1,79 +1,92 @@
+* For more information on caching dependencies, including race conditions, managing caches, expiration, and using cache keys, see the [Caching Dependencies]({{site.baseurl}}/2.0/caching/) page.
 
-For more information see the [Caching Dependencies]({{site.baseurl}}/2.0/caching/) guide.
+### Caching optimization
+{: #cache-optimization }
 
-## Using workspaces
-{: #using-workspaces }
+There are several common ways that your configuration can be optimized to ensure you are getting the most out of your network and storage usage. For example, when looking for opportunities to reduce data usage, consider whether specific usage is providing enough value to be kept. In the cases of caches, this can be quite easy to compare. Does the developer or compute time-saving from the cache outweigh the cost of the download and upload?
 
-![workspaces data flow]( {{ site.baseurl }}/assets/img/docs/workspaces.png)
+Caching optimization strategies can include avoiding unnecessary workflow reruns, combining jobs, creating meaningful workflow orders, and pruning.
 
-**Workspaces are stored for up to 15 days.**
+* For more information on caching optimization and other caching strategies, like partial dependency caching, caching tradeoffs, and using multiple caches, see the [Caching Strategies]({{site.baseurl}}/2.0/caching-strategy/) page.
 
-When a workspace is declared in a job, files and directories can be added to it. Each addition creates a new layer in the workspace filesystem. Downstream jobs can then use this workspace for their own needs or add more layers on top.
+## Workspaces
+{: #workspaces }
 
-Workspaces are not shared between pipeline runs. The only time a workspace can be accessed after the pipeline has run is when a workflow is rerun within the 15
-day limit.
+Workspaces are used to transfer data to downstream jobs as the workflow progresses. When a workspace is declared in a job, files and directories can be added to it. Each addition creates a new layer in the workspace filesystem. Downstream jobs can then use this workspace for their own needs, or add more layers on top.
 
-For more information on using workspaces to persist data throughout a workflow, see the [Workflows]({{site.baseurl}}/2.0/workflows/#using-workspaces-to-share-data-among-jobs) guide. Also see the [Deep Diving into CircleCI Workspaces](https://circleci.com/blog/deep-diving-into-circleci-workspaces/) blog post.
+### Workspace optimization
+{: #workspace-optimization }
 
-## Using artifacts
-{: #using-artifacts }
+If you notice your workspace usage is high and would like to reduce it, try searching for the `persist_to_workspace` command in your `.circleci/config.yml` file to find all jobs utilizing workspaces and determine if all items in the path are necessary. In the cases of caches and workspaces, this can be quite easy to compare. Does the developer or compute time-saving from the cache outweigh the cost of the download and upload?
```

Comment:
Do we also use "cache" in the context of workspaces?

Reply:
I'm not quite sure what you're asking. This is referencing the use of artifacts for saving any caches or workspaces, and whether or not the compute used for them is worth it.

Reply:
Sorry, should've specified: in the "Caching optimization" subsection above this, we're also saying `In the cases of caches, this can be quite easy to compare. Does the developer or compute time-saving from the cache outweigh the cost of the download and upload?` (L33) So I wanted to confirm that repeating that was intended, or if `workspaces` should be added in there somewhere..?

Reply:
Lol sorry, the deleted "using artifacts" title was throwing me off and I was reading this in the context of the wrong section 🤦‍♀️

Reply:
I'm glad you spotted this because it was a mistake of copying a sentence to move it instead of cutting a sentence. I removed the duplicate mention of this from the workspaces section.